src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,25 @@
+package calculator;
+
+import java.util.Arrays;
+
+import static calculator.Separator.separate;
+import static util.StringUtils.containsNegativeNumber;
+import static util.StringUtils.isNullOrEmpty;
+
+
+public class StringCalculator {
+
+    private StringCalculator() {}
+
+    public static int calculate(String expression) {
+        if (isNullOrEmpty(expression)) return 0;
```

Comment:
매직넘버에 대한 의견은 위에서 드렸고 이 부분도 이번엔 코드컨벤션을 준수하면서 연습해보시면 어떨까요?? 😄 
```suggestion
        if (isNullOrEmpty(expression)) { 
            return ZERO;
        }
```
> https://google.github.io/styleguide/javaguide.html#s4.1-braces

Reply:
매직 넘버, 코드 컨벤션도 준수하도록 노력하겠습니다.🔥🔥
감사합니다! 🙏